### ISSUE TITLE
svcscan -v fails with --output=greptext

### ISSUE THREAD

**Issue by exp0se** [volatilityfoundation/volatility#354]
However, without -v its working.
```
$ volatility -f Downloads/zeus.vmem --profile=WinXPSP2x86 svcscan -v --output=greptext
Volatility Foundation Volatility Framework 2.5
Offset|Order|Start|PID|ServiceName|DisplayName|ServiceType|State|BinaryPath|ServiceDll|ImagePath
Traceback (most recent call last):
  File "/usr/bin/volatility", line 192, in <module>
    main()
  File "/usr/bin/volatility", line 183, in main
    command.execute()
  File "/usr/lib/python2.7/dist-packages/volatility/commands.py", line 145, in execute
    func(outfd, data)
  File "/usr/lib/python2.7/dist-packages/volatility/commands.py", line 284, in render_greptext
    self._render(outfd, GrepTextRenderer(self.text_cell_renderers, sort_column = self.text_sort_column), data)
  File "/usr/lib/python2.7/dist-packages/volatility/commands.py", line 274, in _render
    renderer.render(outfd, output)
  File "/usr/lib/python2.7/dist-packages/volatility/renderers/text.py", line 202, in render
    grid.populate(print_row, outfd)
  File "/usr/lib/python2.7/dist-packages/volatility/renderers/__init__.py", line 149, in populate
    for (level, item) in self._generator:
  File "/usr/lib/python2.7/dist-packages/volatility/plugins/malware/svcscan.py", line 498, in generator
    str(val[0] if vals else ""),
NameError: global name 'val' is not defined
```

**Comment by jstore** [comment]
I've run into the same issue. I would really like the combination of -v and greptext to work since the ImagePath is only included with -v and that's useful information for quickly finding malware.

**Comment by gleeda** [comment]
You are using an older version of volatility.  This error was fixed in f7951f4d7c8382f6e80f3f401a124f3a9a7154df  Please update
